sdk-base/src/main/java/com/mapbox/maps/plugin/viewport/experimental/state/ViewportState.kt

```diff
@@ -0,0 +1,28 @@
+package com.mapbox.maps.plugin.viewport.experimental.state
+
+import com.mapbox.maps.plugin.animation.Cancelable
+import com.mapbox.maps.plugin.viewport.ViewportPlugin
+
+/**
+ * Definition of [ViewportPlugin] states.
+ *
+ * [ViewportState] is used to observe the data source associated with the state and query the camera
+ * to keep updating according to current viewport data.
+ */
+interface ViewportState {
+  /**
+   * Observer the new camera options produced by the [ViewportState], it can be used to get the
```

Comment:
`Observer of the new` or `Observe the new` ?

Reply:
Fixed